app/javascript/mastodon/features/hashtag_timeline/components/hashtag_header.jsx

```diff
@@ -4,12 +4,17 @@ import { defineMessages, injectIntl, FormattedMessage } from 'react-intl';
 
 import ImmutablePropTypes from 'react-immutable-proptypes';
 
+import MoreHorizIcon from '@/material-icons/400-24px/more_horiz.svg?react';
 import { Button } from 'mastodon/components/button';
 import { ShortNumber } from 'mastodon/components/short_number';
+import DropdownMenuContainer from 'mastodon/containers/dropdown_menu_container';
+import { withIdentity } from 'mastodon/identity_context';
+import { PERMISSION_MANAGE_TAXONOMIES } from 'mastodon/permissions';
 
 const messages = defineMessages({
   followHashtag: { id: 'hashtag.follow', defaultMessage: 'Follow hashtag' },
   unfollowHashtag: { id: 'hashtag.unfollow', defaultMessage: 'Unfollow hashtag' },
+  admin_moderation: { id: 'hashtag.admin_moderation', defaultMessage: 'Open moderation interface for #{name}' },
```

Comment:
This follows the casing in the account header component: https://github.com/mastodon/mastodon/blob/d3f504245cab5a9a0e89262e0a1398d035dffac9/app/javascript/mastodon/features/account/components/header.jsx

(I'm not sure if camalCase is preferred or not though)

Reply:
Please prefer camel case here

Reply:
Done, I think: https://github.com/mastodon/mastodon/pull/31448/commits/e2f1d45bd0d475bf73c475021350b16b6b6789aa